src/changelog.js

```diff
@@ -8,21 +8,28 @@ let spawnSync = require('child_process').spawnSync;
 let {filter, clone} = require('./utils')
 let Logger = require('./logger')
 
-let github = new GithubApi({
-  version: '3.0.0',
-  timeout: 10000,
-  protocol: 'https'
-});
+function authenticate({host, pathPrefix} = {}) {
```

Comment:
Rather than using a global unic wrapper, creates during authentication, that allows to configure host